staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/types.go

```diff
@@ -61,8 +61,47 @@ type APIServiceSpec struct {
 	Priority int64
 }
 
+type ConditionStatus string
+
+// These are valid condition statuses. "ConditionTrue" means a resource is in the condition;
+// "ConditionFalse" means a resource is not in the condition; "ConditionUnknown" means kubernetes
+// can't decide if a resource is in the condition or not. In the future, we could add other
+// intermediate conditions, e.g. ConditionDegraded.
+const (
+	ConditionTrue    ConditionStatus = "True"
+	ConditionFalse   ConditionStatus = "False"
+	ConditionUnknown ConditionStatus = "Unknown"
```

Comment:
> Are missing conditions treated as Unknown, or is that decided per condition type?

There is no guarantee that every condition will be present and there is no defaulting on these conditions..  How a client uses the data is up to them.  There is a difference between "Unknown" and `<missing>`, so hopefully clients know which one they want to test for.